src/types/xlsx.ts

```diff
@@ -158,6 +158,7 @@ export interface XLSXExportXMLFile {
 export interface XLSXExportImageFile {
   path: string;
   imagePath: string;
+  contentType?: string;
```

Comment:
Was already like that before this PR, but it's unclear what are `path` and `imagePath`. Woudl deserve better name and/or a bit of docstring.

Reply:
It really does. I don't know the difference